data/processors/sections.py

```diff
@@ -44,6 +212,9 @@ def _gen_computed_props(_id, entry, props):
         _append_if_present(props["ids"], computed, "roomcode", _("Raumkennung"))
         if "arch_name" in props["ids"]:
             computed.append({_("Architekten-Name"): props["ids"]["arch_name"].split("@")[0]})
+    if "floor" in props:
+        floor = props["floor"]
+        computed.append({_("Stockwerk"): f"{floor['floor']} (" + floor["name"] + ")"})
```

Comment:
![image](https://user-images.githubusercontent.com/7429408/218742588-10bbc361-7ff7-4879-be77-2ebffae5e8cd.png)

This doesn't work unfortunately because `floor["name"]` is a TranslatableStr. That's why I needed to add the `__add__()` to it.